Rakefile

```diff
@@ -545,6 +547,7 @@ namespace :kokoro do
   desc "Generate configs for kokoro"
   task :build do
     generate_kokoro_configs
+    update_ruby_versions
```

Comment:
```suggestion
    update_kokoro_ruby_versions
```

Reply:
Since I'm now also updating the readme, are you alright with me leaving this as is?

Reply:
I think this needs method name is not very descriptive of what it does. I think it needs to be renamed and/or broken up into smaller methods.

I don't know what to think about having a Kokoro configuration affecting the README.

Reply:
I split them out into separate methods.